.github/workflows/draft-release.yml

```diff
@@ -92,12 +94,12 @@ jobs:
         with:
           github-token: '${{ secrets.ACTIONS_BOT_TOKEN }}'
           script: |-
-            const sha = `${{ github.sha }}`;
-            const tag = `v${{ steps.build.outputs.new_version }}`;
-            const baseBranch = `${{ github.ref_name }}`;
-            const repoURL = `${{ github.server_url }}/${{ github.repository }}`;
-            const prBranch = `${{ env.PR_BRANCH }}`;
-            const prBody = `${{ steps.release_notes.outputs.result }}`;
+            const sha = "${{ github.sha }}";
+            const tag = "v${{ steps.build.outputs.new_version }}";
+            const baseBranch = "${{ github.ref_name }}";
+            const repoURL = "${{ github.server_url }}/${{ github.repository }}";
+            const prBranch = "${{ env.PR_BRANCH }}";
```

Comment:
Instead of interpolating these from GitHub Actions YAML, can we query them directly from the [context](https://github.com/actions/github-script)?

```node
const sha = context.sha
```

I think all the `${{ github.X }}` can be replaced with `context.X`.

Reply:
Most but not all: https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts

I will update what I can.